client/blocks/product-purchase-features-list/sell-online-paypal.jsx

```diff
@@ -21,9 +21,9 @@ export default localize( ( { isJetpack, translate } ) => {
 	return (
 		<div className="product-purchase-features-list__item">
 			<PurchaseDetail
-				buttonText={ translate( 'Start earning' ) }
+				buttonText={ translate( 'Sell online' ) }
 				description={ translate(
-					'Add a PayPal payment button and start selling your goods and services.'
+					'Add a payment button to any post or page to collect PayPal payments for physical products, digital goods, services, or donations.'
```

Comment:
This is great the way that it lists a variety of concrete examples suitable for more folks. Especially good to mention "donations" I think.